internal/attestation/attestation.go

```diff
@@ -38,3 +47,146 @@ type Attestation interface {
 	Statement() []byte
 	Signatures() []signature.EntitySignature
 }
+
+// Extract the payload from a DSSE signature OCI layer
+func PayloadFromSig(sig oci.Signature) (cosign.AttestationPayload, error) {
```

Comment:
Perhaps could be private?
```suggestion
func payloadFromSig(sig oci.Signature) (cosign.AttestationPayload, error) {
```

Reply:
You're right, thanks. Done in upcoming revision.